packages/dd-trace/src/plugin_manager.js

```diff
@@ -107,6 +107,7 @@ module.exports = class PluginManager {
 
     for (const name in pluginClasses) {
       this.loadPlugin(name)
+      this._pluginsByName[name]._tracerConfig = config
```

Comment:
This is becoming pretty hackish, and I still don't see the benefit of having this in the first place, especially since it's only used in a single place which does things differently than all other plugins.

Reply:
Yes, the plugin manager is a mountain of hacky patches and needs to be completely rebuilt